### ISSUE TITLE
Problems with version 0.1.0

### ISSUE THREAD

**Issue by suay1113** [qubvel/segmentation_models.pytorch#119]
Thank you for your work. 
Recently, I was doing a series of comparative experiments, and found that the effect of version 0.1.0 is not as good as that of version 0.0.3.

For example, on my own dataset, the kappa coefficient of LinkNet with ResNet50 is about 0.65 in version 0.0.3, while dropped to 0.5 in version 0.1.0. The same problem occurs on different backbones with the UNet.
I checked the same model established by two versions, and the number of layers was the same but the name of layers were different.
Do you have this problem? If not, I may need to check my code further

**Comment by qubvel** [comment]
Hi, behaviour of final activation is changed! Check that you consider this issue. Did you use any activation function while model initialization?

**Comment by qubvel** [comment]
It would be nice if you provide code for model initialization and loss/metric definition

**Comment by suay1113** [comment]
Here are the code I use.

ENCODER = 'resnet50'
model = smp.Linknet(
    encoder_name=ENCODER, 
    encoder_weights='imagenet', 
    classes=16, 
    activation='softmax',
)

optimizer = optim.Adam(model.parameters(), lr, weight_decay=1e-4, amsgrad=True)
criterion = nn.CrossEntropyLoss(reduction='sum', size_average=True).to(device)

**Comment by qubvel** [comment]
In version 0.0.3 activation is used only for inference (predict method)
In version 0.1.0 activation is used both in train and inference stages.

Did your loss function expect logits or sofrmax input? Make sure you did not do activation two times!

**Comment by suay1113** [comment]
Thank you for your answer. I'll remove the activation during training.
